template.go

```diff
@@ -336,34 +292,37 @@ footer {
 						</td>
 						<td>&mdash;</td>
 						<td class="hideable">&mdash;</td>
+						<td class="hideable">&mdash;</td>
 						<td class="hideable"></td>
 					</tr>
 					{{- end}}
-					{{ range $name := SemSort $ .Dir.Folders }}
+					{{ range $dir := .Dir.RenderedDirs }}
 						<tr class="file">
 							<td></td>
 							<td>
-								<a href="{{ html (PathJoin $.Dir.Path $name) }}">
-									<svg width="1.5em" height="1em" version="1.1" viewBox="0 0 317 259"><use xlink:href="#folder"></use></svg>
-									<span class="name">{{ html $name }}</span>
+								<a href="{{ html (PathJoin $.Dir.Path $dir.Name) }}">
```

Comment:
```suggestion
								<a href="{{ html (PathJoin $.Dir.Path $dir.Name) }}"{{ if ne $dir.Description "" }} title="{{ $dir.Description }}{{- end}}">
```
This gives a title attribute for better accessibility.